src/components/FilterSelectorEntry.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {MouseEvent} from 'react';
+import "../stylesheets/filterbuttons.scss"
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {FilterButtonProps} from "./FilterButton";
+import {IconProp} from "@fortawesome/fontawesome-svg-core";
+
+
+export interface FilterSelectorEntryProps {
+    text: string,
+    icon: IconProp,
+    onClick: (text: string) => void
+}
+
+
+export class FilterSelectorEntry extends React.Component<FilterSelectorEntryProps> {
```

Comment:
Is the selectors list supposed to disappear after you click on a selector?

Reply:
Yes, should be handled using handleClickOutside(). When we actually code the onClick I will add this in.